hadoop-tools/hadoop-azure/src/site/markdown/abfs.md

```diff
@@ -767,9 +767,12 @@ Hflush() being the only documented API that can provide persistent data
 transfer, Flush() also attempting to persist buffered data will lead to
 performance issues.
 
+### <a name="100continueconfigoptions"></a> Hundred Continue Options
+`fs.azure.account.expect.header.enabled`: This configuration parameter is used
+to specify whether you wish to send a expect 100 continue header with each
+append request or not. It is configured to true by default.
```

Comment:
could you add a bit more docs here, as I had to look around to understand this.

proposed
* link to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect
* statement aboubt when it is used
* any docs on your store which cover it
* make clear it is on writing only

```

This flag configures the client to check with the Azure store before uploading a block of data from an output stream. This allows the client to throttle back gracefully -before actually attempting to upload the block.
In experiments this provides significant throughput improvements under heavy load.

```